docs/tildagon-apps/api.md

```diff
@@ -3,6 +3,11 @@ title: Use an API in an app
 weight: 5
 ---
 
+
 !!! tip "You've found a badge CHALLENGE!"
 
     Your challenge, should you choose to accept it, is to create a minimal example app that shows how you can call an API from an app. To see more information and accept the challenge (that is, comment on the issue), see this [issue](https://github.com/emfcamp/badge-2024-documentation/issues/69).
+
+## Example apps
+
+- [A simple example to fetch and display current weather at EMFCamp location](https://github.com/ntflix/Tildagon-Weather).
```

Comment:
This isn't really a self-contained example of how to do this specific thing - in other words, this app does more than just demonstrate how to call an API.

Reply:
Could you let me know what more it does than is necessary for a demo of calling an API please, @hughrawlinson ?

Reply:
Absolutely no need to trim down this is excellent! I'll test it now and add it - for the docs I may merge it into one file but I will do that :)